src/spells.cpp

```diff
@@ -819,17 +833,22 @@ void Spell::postCastSpell(Player* player, bool finishedCast /*= true*/, bool pay
 	}
 
 	if (payCost) {
-		Spell::postCastSpell(player, getManaCost(player), getSoulCost());
+		Spell::postCastSpell(player, getManaCost(player), getHealthCost(player), getSoulCost());
 	}
 }
 
-void Spell::postCastSpell(Player* player, uint32_t manaCost, uint32_t soulCost)
+void Spell::postCastSpell(Player* player, uint32_t manaCost, uint32_t healthCost, uint32_t soulCost)
 {
 	if (manaCost > 0) {
 		player->addManaSpent(manaCost);
 		player->changeMana(-static_cast<int32_t>(manaCost));
 	}
 
+	if (healthCost > 0) {
+		player->addManaSpent(healthCost);
```

Comment:
it shouldn't add mana spent as this could be easily abused, mana is mana and health is health this should stay separated

Reply:
without this using spells what cost health only will not train your magic level at all

Reply:
maybe ill add an option for it in config.lua

Reply:
fixed, added option for it in config.lua